lib/tarantool_server.py

```diff
@@ -307,7 +311,7 @@ def exec_loop(self, ts):
         # one.
         ts.stop_nondefault()
 
-    def killall_servers(self, server, ts, crash_occured):
+    def killall_servers(self, server, ts, crash_occured, timeout_occured):
```

Comment:
You don't use this value anywhere.

Reply:
Sorry, forgot to update the PR from the staging branch.